superset/security/api.py

```diff
@@ -160,3 +163,33 @@ def guest_token(self) -> Response:
             return self.response_400(message=error.message)
         except ValidationError as error:
             return self.response_400(message=error.messages)
+
+    @expose("/custom_login/", methods=["GET"])
+    @event_logger.log_this
+    @safe
+    @permission_name("read")
+    def custom_login(self) -> Response:
```

Comment:
Shouldn't we add a `@protect()` to secure the endpoint ?

Reply:
done